src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -303,15 +304,17 @@ public async Task<Document> ApplyCodeFixesForSpecificDiagnosticIdAsync(Document
                     hasFix: d => this.GetFixableDiagnosticIds(fixer, extensionManager).Contains(d.Id),
                     getFixes: dxs =>
                     {
-                        if (fixAllForInSpan)
+                        using (RoslynEventSource.LogInformationalBlock(FunctionId.CodeFixes_GetCodeFixesAsync, fixer, cancellationToken))
```

Comment:
💡 A using declaration statement would make this a one-line addition to the existing method.

Reply:
Well, I am going to stick with the using statement for consistency with everywhere else in the code where we log telemetry block data. I personally feel that for logging block data using statement is more readable.